tools/ui-components/src/close-button/close-button.tsx

```diff
@@ -21,7 +21,7 @@ export function CloseButton({
   ].join(' ');
   return (
     <button
-      aria-label={label}
+      aria-label={label ?? 'Close'}
```

Comment:
how is this better?

if we have an if statement that assign label value `undefined`, the old way would change the value to `undefined` and be happy, because it's still part of its type.

With this, if ever the value is `undefined`, because the logic we are using in the consumption is faulty. we get `Close`